### DISCUSSION TITLE
About Layerzero sybil reports

### DISCUSSION THREAD

**Discussion by heheboii11** [Cyfrin/foundry-full-course-cu#1841] [General]
Hey everyone, 

  Recently Layerzero announced bounty program for reporting sybils, all these are related to the airdrop from the protocol. I need help regarding proving a report as a false, as it a wrong report but got approved by the team. If there are anyone Who are expertise regarding this matter I would be very happy. Generally first I want to know about Bundlers in ERC3774 standard and how they work. I have some doubts regarding them
```
1. Can a bundler be a normal address or it is a smart contract ? 
2. How does a bundler work ?
3. And I know that bundler combines all the transactions into a single and sends it to mempool( as far I read, I am able to grasp this thing ), so if this is correct, does this means they receive the amount to the address and then execute the transaction ?
 ```
Please help me with this thing @PatrickAlphaC

**Comment by EngrPips** [comment]
Hey @heheboii11, I will need to do a lot of research myself before I can help with this. I don't know much about bundlers myself but let's look forward to what Patrick have to say.

**Comment by PatrickAlphaC** [comment]
Hi! We try to keep discussions here related to the curriculum, not to helping with side projects. Good luck though!

**Comment by heheboii11** [comment]
Hey @PatrickAlphaC, I understand that side projects should not be discussed here. I just want to know more about bundlers so I thought I can post here. Sorry for posting content about other topics

**Reply by EngrPips** [reply]
No worries, man, We are good!
